apps/dashboard/components/agents/agent-details.tsx

```
@@ -22,6 +33,32 @@ export function AgentDetails({
+  const calculateResourceSize = (resource: TypedResource<ResourceType>) => {
+    switch (resource.type) {
+      case 'contract':
+        const value = resource.value as ContractValue;
+        return countBytes(
+          value.abi +
+            value.address +
+            (value.abi || CONTRACT_EXTRA_SIZE_COST) +
+            CONTRACT_EXTRA_SIZE_COST
+        );
+      case 'link':
+        const linkValue = resource.value as LinkValue;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)